learning_resources/views.py

```diff
@@ -64,41 +89,40 @@ def _get_base_queryset(self, resource_type: str = None) -> QuerySet:
             QuerySet of LearningResource objects matching the query parameters
         """
         # Valid fields to filter by, just resource_type for now
-        valid_params = ["resource_type"]
 
-        lr_query = LearningResource.objects.filter(
-            published=True,
-        )
+        lr_query = LearningResource.objects.all()
         query_params_filter = {}
-        for param in self.request.query_params:
-            for valid_param in valid_params:
-                if param.startswith(valid_param):
-                    query_params_filter[param] = self.request.query_params[param]
         if query_params_filter != {}:
             lr_query = lr_query.filter(Q(**query_params_filter))
         if resource_type:
             lr_query = lr_query.filter(resource_type=resource_type)
+
         prefetches = [
             "topics",
             "offered_by",
             "resource_content_tags",
             "runs",
             "runs__instructors",
             "runs__image",
+            "children",
+            "children__child",
+            "children__child__runs",
+            "children__child__runs__instructors",
+            "children__child__course",
+            "children__child__program",
+            "children__child__learning_path",
+            "children__child__department",
+            "children__child__platform",
+            "children__child__topics",
+            "children__child__image",
+            "children__child__offered_by",
+            "children__child__resource_content_tags",
```

Comment:
:scream: We'll need to keep an eye on this

Reply:
Agreed, it's a bit scary how many prefetches are necessary to avoid n+1 errors.  Getting some "Potential unnecessary eager load detected" messages in the logs but I think those are less of a concern?